beater/beater.go

```diff
@@ -713,6 +713,8 @@ func (s *serverRunner) newFinalBatchProcessor(
 	})
 	outputType := monitoring.NewString(monitoring.Default.GetRegistry("libbeat.output"), "type")
 	outputType.Set("elasticsearch")
+	availableBulkIndexers := monitoring.NewInt(monitoring.Default.GetRegistry("libbeat.output"), "availableBulkIndexers")
+	availableBulkIndexers.Set(indexer.Stats().AvailableBulkIndexers)
```

Comment:
This will only set the value of the variable once but won't update its value as it changes over time. That's the reason I used the variable var above.

Reply:
whoops!